cmd/api/src/test/integration/harnesses.go

```diff
@@ -8978,7 +8978,7 @@ func (s *CoerceAndRelayNTLMToSMB) Setup(graphTestContext *GraphTestContext) {
 	graphTestContext.UpdateNode(s.Domain1)
 
 	s.Domain2 = graphTestContext.NewActiveDirectoryDomain("Domain2", domain2Sid, false, true)
-	s.Domain2.Properties.Set(ad.FunctionalLevel.String(), "2016")
+	s.Domain2.Properties.Set(ad.FunctionalLevel.String(), "2008")
```

Comment:
In order to create an edge, the `functionallevel` property must be set to a valid release version before 2012 RC2